[2025-03-30T17:48:06.867+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-30T17:48:06.884+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:47:45.099554+00:00 [queued]>
[2025-03-30T17:48:06.889+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:47:45.099554+00:00 [queued]>
[2025-03-30T17:48:06.889+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-30T17:48:06.906+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_hive_database> on 2025-03-30 17:47:45.099554+00:00
[2025-03-30T17:48:06.913+0000] {standard_task_runner.py:64} INFO - Started process 205 to run task
[2025-03-30T17:48:06.916+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_deliverable', 'create_hive_database', 'manual__2025-03-30T17:47:45.099554+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/dag_deliverable.py', '--cfg-path', '/tmp/tmptga5pj1c']
[2025-03-30T17:48:06.917+0000] {standard_task_runner.py:91} INFO - Job 146: Subtask create_hive_database
[2025-03-30T17:48:06.943+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:47:45.099554+00:00 [running]> on host airflow
[2025-03-30T17:48:06.995+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_deliverable' AIRFLOW_CTX_TASK_ID='create_hive_database' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T17:47:45.099554+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T17:47:45.099554+00:00'
[2025-03-30T17:48:06.995+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-30T17:48:07.011+0000] {base.py:84} INFO - Using connection ID 'hive_default' for task execution.
[2025-03-30T17:48:07.041+0000] {hive.py:475} INFO - USE `default`
[2025-03-30T17:48:07.063+0000] {hive.py:475} INFO - CREATE DATABASE IF NOT EXISTS weather
[2025-03-30T17:48:07.071+0000] {dag_deliverable.py:100} INFO - âœ… Base de datos 'weather' creada correctamente en Hive
[2025-03-30T17:48:07.072+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-30T17:48:07.072+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-30T17:48:07.077+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_deliverable, task_id=create_hive_database, run_id=manual__2025-03-30T17:47:45.099554+00:00, execution_date=20250330T174745, start_date=20250330T174806, end_date=20250330T174807
[2025-03-30T17:48:07.128+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-30T17:48:07.139+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T17:48:07.145+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
